packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -1,9 +1,47 @@
 // This script gets the current recommended `fast` gas price from etherchain
 // to inform the Liquidator and dispute bot of a reasonable gas price to use.
 
-const fetch = require("node-fetch");
+import type { Logger } from "winston";
+import fetch from "node-fetch";
 
-const GAS_ESTIMATOR_MAPPING_BY_NETWORK = {
+interface GasEstimatorMapping {
```

Comment:
including these types of the responses is excellent. we should try get your PR in before https://github.com/UMAprotocol/protocol/pull/3306 to avoid painful rebase.